branchpro/apps/_dash_app.py

```diff
@@ -53,6 +53,134 @@
+                            'font-weight':
+                                'bold',
+                            'cursor':
+                                'pointer'}
+                        ),
+                    dbc.ListGroupItem([
+                        'Incidence Number (compulsory if no ',
+                        html.Span(
+                            'Imported Cases',
+                            id='Imported Cases',
+                            style={
+                                'font-weight':
+                                    'lighter',
+                                'cursor':
+                                    'pointer'}
+                        ),
+                        ' column is present)'],
+                        style={
+                            'font-weight':
+                                'bold',
+                            'cursor':
+                                'pointer'}),
+                    dbc.ListGroupItem(
+                        'Imported Cases (optional)',
+                        style={
+                            'font-weight':
+                                'bold',
+                            'cursor':
+                                'pointer'}),
+                    dbc.ListGroupItem(
+                        'R_t (optional).',
+                        style={
+                            'font-weight':
+                                'bold',
+                            'cursor':
+                                'pointer'})
+                ]),
+            html.P(['e.g.']),
+            dbc.Table.from_dataframe(df, bordered=True, hover=True)]),
+    dbc.ModalFooter(
+                    dbc.Button(
+                        'Close', id='inc_modal_close', className='ml-auto')
+                ),
+]
+
+row1 = html.Tr(
+    [html.Td('0'), html.Td('0'), html.Td('1'), html.Td('0.001')])
+row2 = html.Tr(
+    [html.Td('0.233'), html.Td('2'), html.Td('0'), html.Td('0.003')])
+row3 = html.Tr(
+    [html.Td('0.359'), html.Td('4'), html.Td('0'), html.Td('0.027')])
+row4 = html.Tr([
+    html.Td('0.198'), html.Td('2'), html.Td('0'), html.Td('0.057')])
+
+table_body = [html.Tbody([row1, row2, row3, row4])]
+
+si_modal = [
+    dbc.ModalHeader(html.H6(['Serial Interval'])),
+    dbc.ModalBody(
+        [
+            'The data for the incidences comes in either ',
+            html.Span(
+                '.csv',
+                id='csv_si',
+                style={
+                    'font-weight':
+                        'bold',
+                    'cursor':
+                        'pointer'}
+            ),
+            ' or ',
+            html.Span(
+                '.txt',
+                id='txt_si',
+                style={
+                    'font-weight':
+                        'bold',
+                    'cursor':
+                        'pointer'}
+            ),
+            ' format and will be displayed as a table with no \
+                    columns names.',
```

Comment:
We could repeat that each serial interval is a column (as opposed to a row). It's stated outside perfectly clearly, but it would be nice for the modal to be complete by itself.

Also, we could add a sentence inside the modal elaborating that these could be MCMC samples from a posterior distribution of the serial interval.